recipes/chaiscript/all/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.1.0)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+if (WIN32 AND BUILD_SHARED_LIBS)
```

Comment:
Unneeded, unless you add a `shared` option.

Reply:
Also, `CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS` was introduced in cmake 3.4, so you would either need to increase the version on top, or make a new call here.

Reply:
Ok, I added the shared option and increased the minimum cmake version